src/NuGet.Core/NuGet.Packaging/Signing/SignManifest.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NuGet.Versioning;
+
+namespace NuGet.Packaging.Signing
+{
+    /// <summary>
+    /// Represents the XML manifest containing a set of files and hashes that can be signed.
+    /// </summary>
+    public class SignManifest
```

Comment:
Consider just `Manifest`.  It's already in the `Signing` namespace.

* `Sign` is a verb not an adjective
* `SignManifest` is a better method name than a class name
* Not sure `SignedManifest` will be technically correct

See https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces

Reply:
I'm open to a better name, but I think Manifest is confusing since the nuspec is also called manifest in other places. When exposed on the package readers the different in namespace isn't very clear.

Reply:
How about `SigningManifest`?